examples/san_manager.rb

```diff
@@ -32,7 +32,7 @@
 end
 
 # Example: remove the SAN manager
-oneview_san_manager '172.18.15.1' do
+oneview_san_manager 'Remove 172.18.15.1' do
```

Comment:
`(name: '172.18.15.1')` should not be necessary as long as the `title` is correct, a.k.a. Drop these modifications and it *should* work